### PR TITLE
Modify Release Attempt 2

### PR DESCRIPTION
**Pull request by amagana3**
Finally!! I got it working by doing the following:
- I forked the project
- Setup TravisCI for my personal account using the fork
- Removed all deploy commands and only allowed the `make release` to run.
- Realized midway through that the TravisCI env. is setup as MacOSx and not Linux (angry hand throws here)
- Realized that the `@gpg` command HIDES THE COMMAND... so I was thinking it was failing this whole time (more arm flailing)
- Fixed the real issue - when we signed it was still prompting the passphrase, but I got it working now.

For some odd reason, even though this is an osx environment, it is _not_ the same as my local, but either way this is good to go and we're now ready to start publishing ❗

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+plan:
+	@terraform plan
+
+install:
+	go get -t -v ./...
+
+release:
+package=github.com/Cox-Automotive/terraform-provider-alks
+GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
+
```
Comment (webbbarker) [inline Makefile:23]: Your commit duplicated these lines.

**Review by webbbarker — CHANGES_REQUESTED** [review]
Looks like your commits are duplicating some lines in the Makefile.  Get that cleaned up and I can approve your changes.

File: Makefile (hunk 1 above)
Comment (amagana3) [reply Makefile]: Good catch - hadn't seen that last night

**Review by webbbarker — APPROVED** [review]
👍

**Pull request merged**